tests/modules/i18n/input-method/kime-configuration.nix

```diff
@@ -0,0 +1,12 @@
+{ config, pkgs, ... }:
+
+{
+  i18n.inputMethod = {
+    enabled = "kime";
+    kime.config = { engine = { hangul = { layout = "dubeolsik"; }; }; };
+  };
```

Comment:
Can you add a stub package to prevent fetching the package when running the tests?

``` nix
{
  test.stubs.kime = { };
}
```

Reply:
Thanks for review! I added the stub package https://github.com/nix-community/home-manager/pull/2621/commits/6de7eab29b74107822ad6a74d70cf652efb06883 :)